scripts/prepare-machine.ps1

```diff
@@ -64,6 +64,13 @@ $ClogDownloadUrl = "https://github.com/microsoft/CLOG/releases/download/v$ClogVe
 
 $MessagesAtEnd = New-Object Collections.Generic.List[string]
 
+if ($PSVersionTable.PSVersion.Major -lt 7) {
+
+    Write-Error ("`nPowerShell v7.x is needed for this script to work. " +
+                 "Please visit https://github.com/microsoft/msquic/blob/main/docs/BUILD.md#powershell-usage")
+    exit
```

Comment:
The `exit` is unnecessary because of `$PSDefaultParameterValues['*:ErrorAction'] = 'Stop'` above.

Reply:
```suggestion
```